### PR TITLE
Last week in ManageIQ 22 Aug 2016

### PR DESCRIPTION
**Pull request by juliancheal**

### CODE REVIEW

**Comment by JPrause** [comment]
@imtayadeway @syncrou can you review for tomorrow's blog post. @Fryguy

File: source/blog/2016-08-22-last-week-in-manageiq.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+author: juliancheal
+comments: true
+published: true
+---
+
+What ho! Welcome to another jolly edition of Last Week in ManageIQ. [Julian](https://juliancheal.co.uk) here, hailing from the Providers team. Let's put the kettle on and take a look over what's happened this last week in ManageIQ.
+
+## Featured
+
+We've had a spiffing week with [138 pull requests merged][PRs merged last week] from our marvellous contributors. ManageIQ always welcomes new contributors, we have a great list of
```
Comment (syncrou) [inline source/blog/2016-08-22-last-week-in-manageiq.md]: haha!  -  I was going to make a comment on the "correct" spelling of `marvelous` - BUUT - Then I found [this](http://www.oxforddictionaries.com/us/definition/english/marvellous)

Never mind.  Nothing to see here.

File: source/blog/2016-08-22-last-week-in-manageiq.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+overhauling the MiQ Roles API to:
+> remove unused and needlessly complex code.
+
+As with all good refactorings, this included 48 new additions, but managed to remove 124 lines of code!
+
+## Fixed
+
+### [Improving the API](https://github.com/ManageIQ/manageiq/pull/10632)
+
+Fellow Brit [Time Wade](https://github.com/imtayadeway) has been refactoring away at the API, this time extracting classes to remove the `Settings concern` from every API call. Good job Tim.
```
Comment (syncrou) [inline source/blog/2016-08-22-last-week-in-manageiq.md]: Possibly - s/Time/Tim/ - (though that may be a special English spelling of `Tim` I'm not aware of  😄)

File: source/blog/2016-08-22-last-week-in-manageiq.md (hunk 1 above)
Comment (syncrou) [inline source/blog/2016-08-22-last-week-in-manageiq.md]: The count may be off. I'm seeing 147 with this url. 

Though I know we've had a number of discussions around how to count these 😄  - So please ignore if I'm behind on that front.

**Comment by syncrou** [comment]
@juliancheal - This was a riot to read!  Thanks so much for all your work!  

Just a couple of small suggestions.

File: source/blog/2016-08-22-last-week-in-manageiq.md (hunk 1 above)
Comment (juliancheal) [inline source/blog/2016-08-22-last-week-in-manageiq.md]: I may have used the wrong method of counting :(

File: source/blog/2016-08-22-last-week-in-manageiq.md (hunk 2 above)
Comment (juliancheal) [inline source/blog/2016-08-22-last-week-in-manageiq.md]: Haha, I blame autocorrect ;)

**Comment by JPrause** [comment]
@syncrou Julian made the edits, would you say we're, thumbs up for merge?

**Comment by syncrou** [comment]
👍 LGTM

**Comment by JPrause** [comment]
@chessbyte can you merge for this week's blog entry.

**Comment by juliancheal** [comment]
miq-bot add label "blog post"

File: source/blog/2016-08-22-last-week-in-manageiq.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+## Fixed
+
+### [Improving the API](https://github.com/ManageIQ/manageiq/pull/10632)
+
+Fellow Brit [Tim Wade](https://github.com/imtayadeway) has been refactoring away at the API, this time extracting classes to remove the `Settings concern` from every API call. Good job Tim.
+
+### [Nicer Errors](https://github.com/ManageIQ/manageiq/pull/10511)
+
+Everyone likes error messages, but no one likes badly formatted ones. [Daniel Berger](https://github.com/djberg96) with this PR, formats the `XML` output of `SCVMM` into a nicer string.
```
Comment (chessbyte) [inline source/blog/2016-08-22-last-week-in-manageiq.md]: Which PR?

File: source/blog/2016-08-22-last-week-in-manageiq.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+author: juliancheal
+comments: true
+published: true
+---
+
+What ho! Welcome to another jolly edition of Last Week in ManageIQ. [Julian](https://juliancheal.co.uk) here, hailing from the Providers team. Let's put the kettle on and take a look over what's happened this last week in ManageIQ.
+
+## Featured
+
+We've had a spiffing week with [147 pull requests merged][PRs merged last week] from our marvellous contributors. ManageIQ always welcomes new contributors, we have a great list of
```
Comment (chessbyte) [inline source/blog/2016-08-22-last-week-in-manageiq.md]: The link to 147 PRs comes up with 167 PRs.

File: source/blog/2016-08-22-last-week-in-manageiq.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+A small change, but a useful one. Removing an unused and _broken_ OAuth button. A good spot from [Martin Hradil](https://github.com/himdel)
+
+### [No methods no cry](https://github.com/ManageIQ/manageiq/pull/10547)
+
+[Keenan Brock](https://github.com/kbrock) is fixing relationships this week by removing unused relationship methods. A good example of a `TODO:` comment that never got `TODONE`.
+
+## Wrapping up
+
+Not technically last week, but ManageIQ developer Chris Arcand, made it into [Ruby Weekly](https://chrisarcand.com/the-search-for-class0x0000001ab51700/?utm_source=rubyweekly&utm_medium=email) with `The Search For <Class:0x00000027b59290>`, a thrilling tail of hunting for that mysterious bug in a codebase.
```
Comment (chessbyte) [inline source/blog/2016-08-22-last-week-in-manageiq.md]: typo?  should `tail` be `tale`?

Comment (juliancheal) [inline source/blog/2016-08-22-last-week-in-manageiq.md]: Good spot!

**Comment by Fryguy** [comment]
Doesn't the file title have to change? (i.e. no title collisions or the blog is unhappy)  @chrisarcand @himdel ?

File: source/blog/2016-08-22-last-week-in-manageiq.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+[issues](https://github.com/manageiq/manageiq/issues) so take a gander to see where
+you can help us.
+
+## Improved
+
+### [The Azure, Azure](https://github.com/ManageIQ/manageiq/pull/10629)
+
+[Bronagh Sorota] (https://github.com/bronaghs) has improved the naming of the Azure regions. When creating an Azure provider with no instances; instead of naming the region `Azure-eastus`, we incorrectly called it `Azure-Azure-eastus`
+
+### [MiQ Roles API](https://github.com/ManageIQ/manageiq/pull/10452)
```
Comment (Fryguy) [inline source/blog/2016-08-22-last-week-in-manageiq.md]: MiQ is cased funny...it should be all caps, or spelled out ManageIQ (or just removed entirely)

**Comment by juliancheal** [comment]
> Doesn't the file title have to change? (i.e. no title collisions or the blog is unhappy) @chrisarcand @himdel ?

Isn't the title of this different than all the rest already? As it has the date.

**Comment by chrisarcand** [comment]
@juliancheal No, something weird happens with the current static site builder (I thought the same thing, hence mine is the 'only one allowed to just be last-week-in-manage'). You'll have to add some sort of full title. :/

File: source/blog/2016-08-22-last-week-in-manageiq.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+## New
+
+### [New LDAP Gem](https://github.com/ManageIQ/manageiq/pull/10635)
+
+If you use `ldap` you might be happy to know [Gregg Tanzillo](https://github.com/gtanzillo) has updated us to version `0.14.0`. Fixing some encoding errors with the update :)
+
+### [Active Support for Everyone](https://github.com/ManageIQ/manageiq/pull/10627)
+
+[Nick Carboni](https://github.com/carbonin) has added the ability to use methods such as `Hash#symbolize_keys` and `Time#current` in production, by requiring `active_support/all` and making it safer for everyone needing those methods
```
Comment (Fryguy) [inline source/blog/2016-08-22-last-week-in-manageiq.md]: This is misleading.  He changed it just for the PostgresHAAdmin code (not the whole application :P )

File: source/blog/2016-08-22-last-week-in-manageiq.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+### [No methods no cry](https://github.com/ManageIQ/manageiq/pull/10547)
+
+[Keenan Brock](https://github.com/kbrock) is fixing relationships this week by removing unused relationship methods. A good example of a `TODO:` comment that never got `TODONE`.
+
+## Wrapping up
+
+Not technically last week, but ManageIQ developer Chris Arcand, made it into [Ruby Weekly](https://chrisarcand.com/the-search-for-class0x0000001ab51700/?utm_source=rubyweekly&utm_medium=email) with `The Search For <Class:0x00000027b59290>`, a thrilling tale of hunting for that mysterious bug in a codebase.
+
+As always we have too many PRs to cover here in detail. You are always welcome to have a peak at them [here][PRs merged last week]. And with that pip pip & toodle-oo, and don't forget to look out for the next
```
Comment (Fryguy) [inline source/blog/2016-08-22-last-week-in-manageiq.md]: Typo `peak` -> `peek`  (unless that's some British spelling I'm unawares of :wink:)

File: source/blog/2016-08-22-last-week-in-manageiq.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+A small change, but a useful one. Removing an unused and _broken_ OAuth button. A good spot from [Martin Hradil](https://github.com/himdel)
+
+### [No methods no cry](https://github.com/ManageIQ/manageiq/pull/10547)
+
+[Keenan Brock](https://github.com/kbrock) is fixing relationships this week by removing unused relationship methods. A good example of a `TODO:` comment that never got `TODONE`.
+
+## Wrapping up
+
+Not technically last week, but ManageIQ developer Chris Arcand, made it into [Ruby Weekly](https://chrisarcand.com/the-search-for-class0x0000001ab51700/?utm_source=rubyweekly&utm_medium=email) with `The Search For <Class:0x00000027b59290>`, a thrilling tale of hunting for that mysterious bug in a codebase.
```
Comment (chrisarcand) [inline source/blog/2016-08-22-last-week-in-manageiq.md]: Hah <3

I'd change "that mysterious bug in a codebase." to "a mysterious bug in our codebase." or "a mysterious bug in ManageIQ."

**Comment by JPrause** [comment]
@juliancheal can you rename it to: 2016-08-22-last-week-in-manageiq-the azure, nice errors, and gems.html.md

**Comment by juliancheal** [comment]
Sure thing @JPrause

**Comment by JPrause** [comment]
@chessbyte @Fryguy I believe we're ready for merge.

**Pull request merged**